### ISSUE TITLE
Incorrect appsettings.json generated for Blazor WebAssembly hosted individual auth

### ISSUE THREAD

**Issue by SteveSandersonMS** [dotnet/aspnetcore#21139]
For hosted + individual auth, the user account information is stored inside the application. So it doesn't seem like we need it to generate the `appsettings.json` file that it does:

```json
{
  "Local": {
    "Authority": "https:login.microsoftonline.com/",
    "ClientId": "33333333-3333-3333-33333333333333333"
  },
}
```

Additionally, notice the invalid trailing comma.

**Comment by SteveSandersonMS** [comment]
Also in the same scenario, the template generates the following `appsettings.Development.json` file:

```json
{
   "Local": {
     "Authority": "https:login.microsoftonline.com/",
     "ClientId": "33333333-3333-3333-33333333333333333"
   }
}
```

Likewise, this shouldn't be generated. However another thing to notice is how it differs from the `appsettings.json` file above:

 * No invalid trailing comma
 * Indentation is different (three spaces before `"Local"` instead of two)

**Comment by pranavkm** [comment]
Let's consider this for GA.

**Comment by captainsafia** [comment]
This has been fixed in #21147.
